src/main/cpp/option_processor.cc

```diff
@@ -322,6 +322,9 @@ blaze_exit_code::ExitCode OptionProcessor::GetRcFiles(
       SearchNullaryOption(cmd_line->startup_args, "workspace_rc", true)) {
     const std::string workspaceRcFile =
         blaze_util::JoinPath(workspace, kRcBasename);
+    // Legacy behavior.
+    rc_files.push_back(
+        workspace_layout->GetWorkspaceRcPath(workspace, cmd_line->startup_args));
```

Comment:
the legacy behavior would be to have this hidden behind --master_blazerc, in case that matters to you at this point